docs/run-keep-node.adoc

```diff
@@ -0,0 +1,359 @@
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
+
+= Run Keep Node
+
+[#system-considerations]
+== System Considerations
+
+The Keep Network expects certain capabilities for each node running on the network.
+To help attain these capabilities consider the following criteria:
+
+- It is paramount that Keep nodes remain available to the Keep Network. We strongly encourage a
+  stable and redundant internet connection.
+- A connection to a production grade self-hosted or third party Ethereum node deployment.
+- Persistent and redundant storage that will survive a VM or container rotation, and disk failure.
+- Each node running on the network requires a unique Ethereum operator account.
+- Each node running on the network requires a unique IP address or a unique
+  application port running under the same IP.
+
+=== Recommended Machine Types
+
+Your operating environment will ultimately dictate what machine type to go with.  This is
+particularly relevant if you're running a containerized solution where multiple applications
+are sharing VM resources.  The below types are sufficient for running at least one instance of the
+Keep Node.
+
+// TODO: Replace TBDs with recommendations.
```

Comment:
Is it something we are planning to do in this PR? I think the same recommendations as for v1 ECDSA are sufficient.

Reply:
I used the ECDSA V1 recommendations for now e53ecd449d44742e5e1fef7abd79cd9ad2de9f13